components/menu/index.css

```
@@ -291,17 +291,46 @@
 	}
 }
 
-.spectrum-Menu-itemIcon {
+.spectrum-Menu-itemIcon.spectrum-Icon {
```

```suggestion
.spectrum-Menu-itemIcon.spectrum-Icon {
```

I'm not sure this is going to translate to SWC since the icon will be pulled in as a nested web component in the shadow dom.  That's why we've moved generally toward not nesting the class name of components and using mods passthroughs instead. If icon is overriding these styles do we need to use a fill and color passthrough?

---

We still need to increase the specificity in order to solve the source order issue. Using mod passthroughs would not address that. The issue stems from the same specificity of the single class names like this one, and `.spectrum-Icon`. And if the icon CSS is included afterwards, it is overriding things it shouldn't. 

What do you think about using `.spectrum-Menu .spectrum-Menu-itemIcon` instead?

---

I'll work on a different method using mod passthroughs. It'll need an additional mod or two added to Icon, but I think it will work.

---

I've updated this to use `.spectrum-Menu .thing` instead of `.thing.spectrum-Icon`, as SWC should convert that just fine, correct? That seems in line with most other CSS and doesn't require having to add a mod to the icon component for every property that gets changed by menu's icon classes, now and into the future.
2ec451d2c85c3e1e7973b7aa29de1c8da2097359